### DISCUSSION TITLE
How to be compatible with old version data?

### DISCUSSION THREAD

**Discussion by sivanzheng** [verdaccio/verdaccio#4782] [Q&A] [answered]
I upgraded from version 3 to version 5, and then I couldn't search for the package. I found that the storage data was incompatible. There was a .verdaccio-db in version 5. Is this the reason? Hope to get your answer.

**Comment by juanpicado** [comment, marked as answer]
> There was a .verdaccio-db in version 5. Is this the reason?

Probably yes, database did change the name probably, there is an article at the blog or read the docs in the configuration part, probably you will find some answers. But the internal structure of the database hasn't changed.

**Reply by sivanzheng** [reply]
Thanks, I created the db file manually according to the article, it worked!
